### PR TITLE
Added Error page to ServerSide Blazor (#11243)

### PR DESCRIPTION
**Pull request by isaacrlevin**
Summary of the changes
 - Added Error page to Server Blazor template

Addresses #11766

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Error.razor [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿@page "/error"
```
Comment (ryanbrandenburg) [inline src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Error.razor]: Update https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/test/template-baselines.json to reflect this new file.

**Review by ryanbrandenburg — APPROVED** [review]
Looks like :shipit: once the tests pass.

File: src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Error.razor (hunk 1 above)
Comment (isaac2004) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Error.razor]: Didn't seem to work

Comment (ryanbrandenburg) [reply src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Error.razor]: The `blazorserver` entry there has 4 entries, you only updated one.

**Comment by isaac2004** [comment]
@ryanbrandenburg :shipit:

**Pull request merged**